doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+   Only use this legacy variant if your network does not support ePCO.
+
+#. Click the :guilabel:`Erase & write` button in the :guilabel:`Device` section to program the DK.
+   Do not unplug or turn off the DK during this process.
+
+   .. figure:: /images/programmer_erasewrite_nrf9160dk.png
+      :alt: Programmer - Erase & write
+
+      Programmer - Erase & write
+
+If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (Cmd+R on macOS), and try again.
+If you continue to updating the modem firware, you can go directly to Step 5.
+
+.. _nrf9160_gs_updating_fw_modem:
+
+Updating the modem firmware
+===========================
+
+To update the modem firmware, complete the following steps:
+
+1. Open the Programmer app.
+#. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF91**.
+   On DK v0.9.0 and earlier, this is the **SW5** switch.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and then turn the DK on.
+#. Click :guilabel:`Select device` and select the DK from the drop-down list.
+   You can identify the nRF9160 DK by the fact that it has three COM ports.
+
+   .. figure:: /images/programmer_com_ports.png
+      :alt: Programmer - COM ports
+
+      Programmer - COM ports
+
+   If the three COM ports are not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
+
+   The button text changes to the type of the selected device, with the SEGGER ID below the name.
+   The :guilabel:`Device Memory Layout` section also changes its name to the device name, and indicates that the device is connected.
+   If the :guilabel:`Auto read memory` option is selected in the :guilabel:`Device` section ofthe side panel, the memory layout will update.
+   If it is not selected and you wish to see the memory layout, click :guilabel:`Read` in the :guilabel:`Device` section of the side panel.
+
+#. Click :guilabel:`Add file` in the :guilabel:`File` section of the side panel.
+
+   .. figure:: /images/programmer_addfile_nrf9160dk.png
+      :alt: Programmer - Add file
+
+      Programmer - Add file
+
+#. Navigate to where you extracted the firmware, and choose the :file:`mfwnrf9160_<version-number>.zip` file.
+#. Click :guilabel:`Write` in the :guilabel:`Device` section of the side panel.
+
+   .. figure:: /images/programmer_write_nrf9160dk.png
+      :alt: Programmer - Write
+
+      Programmer - Write
+
+   The :guilabel:`Modem DFU` dialog window appears.
+
+   .. figure:: images/programmerapp_modemdfu.png
+      :alt: Modem DFU window
+
+      The Modem DFU window
+
+#. Click the :guilabel:`Write` button in the :guilabel:`Modem DFU` dialog window to update the firmware.
+   Do not unplug or turn off the device during this process.
+
+When the update is complete, you see a success message.
+
+If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (Cmd+R on macOS), and try again.
```

Comment:
Should this be part of the following note? They could be phrased similarly and maybe add bullets inside the note (not sure if it's allowed in ncs), one for restarting Programmer and one for erase all.

Reply:
Erasing all data seems like a very drastic measure, and I wouldn't want to imply that restarting Programmer is an equivalent option.

Conversely, should the note be made more specific with "If you experience significant problems [...]"?

Reply:
The point is to apply similar formatting for similar type of information for consistency (see RMF p. 82 and https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Style+and+usage+guide). If these are not same-level notes, maybe erase all should be a caution or warning.

Reply:
Should line 126, containing the same note about restarting Programmer, then also be formatted as a note? It is the exact same information as the first line here.

I would personally prefer moving lines 126 and 182 to a different part of the respective sections instead of combining line 182 with lines 186-188. A place for those lines could be the start of the respective sections.